cmd/adapter/adapter.go

```diff
@@ -275,6 +295,8 @@ func main() {
 	cmd := &PrometheusAdapter{
 		PrometheusURL:         "https://localhost",
 		MetricsRelistInterval: 10 * time.Minute,
+		MetricsHost:           "0.0.0.0",
```

Comment:
The `::` wildcard might be a better default since it would work for both ipv4 and ipv6.

Reply:
Done